go.mod

```diff
@@ -44,6 +43,7 @@ require (
 )
 
 require (
+	github.com/avast/retry-go v3.0.0+incompatible
```

Comment:
Instead of introducing a new one can we use this from k8s client https://pkg.go.dev/k8s.io/client-go/util/retry
We use this in flyteadmin too

Reply:
not sure why the mod file changed.  this retry library was already in [here](https://github.com/flyteorg/flytectl/blob/master/pkg/sandbox/start.go#L12) and has been there for a while.  Can we keep it for now.

Reply:
Sure